src/model/statistics/EntityItem.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <QDebug>
+
+#include <fastdds-monitor/backend/backend_utils.h>
+#include <fastdds-monitor/model/statistics/EntityItem.h>
+
+namespace models {
+
+QVariant EntityItem::data(
+        int role) const
+{
+    switch (role)
+    {
+        case idRole:
+            return this->entity_id();
+        case nameRole:
+            return this->name();
+        case kindRole:
+            return this->kind();
+        case nameIdRole:
+            return this->nameId();
+        default:
+            return QVariant();
+    }
+}
+
+QString EntityItem::nameId() const
+{
+    QString nameId = name();
+    nameId.append("-");
```

Comment:
I would suggest a way of merge name and string with a char or format that a simple name could not have, for example `name:<id>`

Reply:
Do you mean `<name>:<id>` or `name:<id>`?

Reply:
Second one.
`<name>:'<'<id>'>'`